diversity-lunch-be/src/test/java/de/brockhausag/diversitylunchspringboot/integrationstests/EMailControllerIT.java

```diff
@@ -131,14 +131,33 @@ public void testSendTestMail_expectedCorrectTestContentInMail() {
 
     @SneakyThrows
     @Test
-    public void testSendTestMailToUser_expectedToNotThrowException() {
+    public void testSendTestMailToUser_withValidId_expectedOkStatus() {
+        when(this.diversityLunchMailProperties.getSender()).thenReturn("diversitylunchtest@brockhaus-ag.de");
         ProfileEntity tmpProfileEntity = profileTestdataFactory.createEntity();
-        var accountEntity1 = accountService.getOrCreateAccount(tmpProfileEntity.getEmail());
-        long id = 42;
+        AccountEntity accountEntity1 = accountService.getOrCreateAccount(tmpProfileEntity.getEmail());
+        //ProfileEntity is needed, please don't remove it, although IntelliJ suggests to do so.
+        ProfileEntity profileEntity1 = profileService.createProfile(tmpProfileEntity, accountEntity1.getId()).orElseThrow();
+        long id = tmpProfileEntity.getId();
 
-        mockMvc.perform(
+        mockMvcSecurity.perform(
                 post("/mailing/sendTestMailToUser?id=" + id)
                         .header(HttpHeaders.AUTHORIZATION, "Bearer " + profileTestdataFactory.getTokenStringFromId(accountEntity1.getUniqueName()))
         ).andExpect(status().isOk());
     }
+
+    @SneakyThrows
+    @Test
+    public void testSendTestMailToUser_withInvalidId_expectedForbiddenStatus() {
+        when(this.diversityLunchMailProperties.getSender()).thenReturn("diversitylunchtest@brockhaus-ag.de");
+        ProfileEntity tmpProfileEntity = profileTestdataFactory.createEntity();
+        var accountEntity1 = accountService.getOrCreateAccount(tmpProfileEntity.getEmail());
+        //ProfileEntity is needed, please don't remove it, although IntelliJ suggests to do so.
+        var profileEntity1 = profileService.createProfile(tmpProfileEntity, accountEntity1.getId()).orElseThrow();
```

Comment:
Variable could be removed, not needed here

Reply:
Which variable do you want to be removed?

Reply:
"profileEntity1 "